File: runner/questions/c-smoke/TestRunner.c
#include <stdio.h>

// __STUDENT_CODE__

int main(void) {
    int passed = 0;
    int total = 4;
    if (double_value(0) == 0) passed++;
    if (double_value(2) == 4) passed++;
    if (double_value(-3) == -6) passed++;
    if (double_value(1000000) == 2000000) passed++;
    printf("RESULT %d/%d\n", passed, total);
    return passed == total ? 0 : 1;
}

File: runner/questions/py-smoke/TestRunner.py
def check(label, expected, actual):
    if expected == actual:
        print(f"PASS {label} expected={expected!r} actual={actual!r}")
        return 1
    print(f"FAIL {label} expected={expected!r} actual={actual!r}")
    return 0


// __STUDENT_CODE__

passed = 0
total = 2
passed += check("double small", 8, double(4))
passed += check("double negative", -6, double(-3))
print(f"RESULT {passed}/{total}")
